plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeColumnHandle.java

```diff
@@ -92,6 +96,12 @@ public boolean equals(Object obj)
                 this.columnType == other.columnType;
     }
 
+    public long getRetainedSizeInBytes()
+    {
+        // baseType is not accounted for as the instances are cached (by TypeRegistry) and shared
```

Comment:
who is `baseType`? did you mean `type`?

nit: include DeltaLakeColumnType

Reply:
> who is `baseType`? did you mean `type`?

👍 
 
> nit: include DeltaLakeColumnType

what is there to account for? It is an enum.